relays/client-pangolin/src/lib.rs

```diff
@@ -0,0 +1,108 @@
+use std::time::Duration;
+
+// -- s2s bridger ---
+use codec::Encode;
+use sp_core::{Pair, storage::StorageKey};
+use sp_runtime::{
+	generic::SignedPayload,
+	traits::IdentifyAccount,
+};
+
+use relay_substrate_client::{Chain, ChainBase, ChainWithBalances, TransactionSignScheme};
+
+// --- darwinia ---
+use pangolin_runtime::*;
+use frame_support::traits::Len;
+
+/// Pangolin header id.
+pub type HeaderId = relay_utils::HeaderId<drml_primitives::Hash, drml_primitives::BlockNumber>;
+
+
+/// Rialto signing params.
+pub type SigningParams = sp_core::sr25519::Pair;
+
+/// Rialto header type used in headers sync.
+pub type SyncHeader = relay_substrate_client::SyncHeader<drml_primitives::Header>;
+
+
+/// Millau chain definition.
+#[derive(Debug, Clone, Copy)]
+pub struct PangolinRelayChain;
+
+impl ChainBase for PangolinRelayChain {
+	type BlockNumber = drml_primitives::BlockNumber;
+	type Hash = drml_primitives::Hash;
+	type Hasher = drml_primitives::Hashing;
+	type Header = drml_primitives::Header;
+}
+
+impl Chain for PangolinRelayChain {
+	const NAME: &'static str = "Pangolin";
+	const AVERAGE_BLOCK_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
Pangolin's block interval is 6

Reply:
https://github.com/darwinia-network/bridger/commit/fb7851a7a7947152642820ba0b5e458d0d993520